src/values/ObjectValue.js

```diff
@@ -530,7 +530,7 @@ export default class ObjectValue extends ConcreteValue {
     });
   }
 
-  getOwnPropertyKeysArray(): Array<string> {
+  getOwnPropertyKeysArray(allowAbstractKeys: ?true): Array<string> {
```

Comment:
This would allow for `null`. I prefer `allowAbstractKeys?: true`, or `: boolean = false`.